usvm-ts/src/test/kotlin/org/usvm/util/TsMethodTestRunner.kt

```diff
@@ -215,7 +216,8 @@ abstract class TsMethodTestRunner : TestRunner<TsTest, EtsMethod, EtsType?, TsMe
     }
 
     override val runner: (EtsMethod, UMachineOptions) -> List<TsTest> = { method, options ->
-        TsMachine(scene, options).use { machine ->
+        val tsMachineOptions = TsOptions()
+        TsMachine(scene, options, tsMachineOptions).use { machine ->
```

Comment:
Why two sets of options now? Can't we inherit UMachineOptions?

Reply:
Because umachineoptions is final data class